docs/api.md

```diff
@@ -259,9 +293,9 @@ GET /api/emotes
 
 <a name='new-emote'></a>
 ### Add a new emote [POST /api/emotes]
-+ requires [permission](#permissions): MANAGE_EMOTES
++ requires [permission](#permissions) `manageEmotes`
 + `imageURL` (string)
-+ `shortcode` (Name) - Should not include colons (`:`).
++ `shortcode` (Name) - Should not include colons (`:`). Must be unique, even if the user has the `allowNonUnique` [permission](#permissions).
```

Comment:
Can you make `shortcode` a `string`, not a name?
It needs to fit /^[^ :]+$/ - ie. any chars but spaces and colons,